AirCasting/CreateSessionViews/SelectPeripheralView.swift

```diff
@@ -61,16 +61,22 @@ struct SelectPeripheralView: View {
                     } else {
                         connectButton.disabled(true)
                     }
-                }
+                }.alert(isPresented: $showBluetoothAlert, content: {
+                    Alert(title: Text(Strings.SelectPeripheralView.alertTitle), message: Text(Strings.SelectPeripheralView.alertMessage), dismissButton: .default(Text(Strings.SelectPeripheralView.alertAccept)))
+                })
                 .padding()
                 .frame(maxWidth: .infinity, minHeight: geometry.size.height, alignment: .top)
             }
-        }
-        
+        }.onChange(of: bluetoothManager.isConnected, perform: { value in
+            showBluetoothAlert = !value
+        })
+        .onAppear(perform: {
+            bluetoothManager.isConnected = true
```

Comment:
why is this here? It seems unrelated and if needed I feel we need another way since this won't be clear to readers (also we should not use BluetoothManager in View objects). Can you elaborate?

Reply:
I think the reason that the BT manager is here is not given by us. Mostly it was done by someone else. I am a bit confused now, going to try to delete .onApper but the rest will stay as if it is not mine

Reply:
All right 👌